lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -238,6 +238,9 @@ def self.teardown_drb_for_ruby_method
       DRb.stop_service
       # Set the ID conv to nil so that the cache can be GC'ed
       DRb.install_id_conv(nil)
+      holder = @@global_id_conv.instance_variable_get('@holder')
+      holder.instance_variable_get('@keeper').try(:kill) if holder
```

Comment:
Please link to [this for context](https://github.com/ruby/ruby/blob/8ef6dacb248876b444595a26ea78c35eb07a188b/lib/drb/timeridconv.rb#L28) and [this](https://bugs.ruby-lang.org/issues/12342) so we can clean this up if they fix it in 2.3.2 or later.  Please put a comment about what we're doing because it's not obvious at all.